src/content/developers/docs/ides/index.md

```diff
@@ -57,4 +63,6 @@ Most established IDEs have built plugins to enhance the Ethereum development exp
 
 ## Further reading {#further-reading}
 
+- [Blockchain development tools](https://www.alchemy.com/overviews/20-blockchain-development-tools) - Alchemy's top 20 blockchain development tools
```

Comment:
While seems like an ok list, this doesn't seem all that relevant to people looking at IDEs 🤔 

```suggestion
```